examples/pytorch/Logging/pytorch log model.ipynb

```diff
@@ -0,0 +1,416 @@
+    "        x = self.fc2(x)\n",
+    "        return x\n",
+    "\n",
+    "model = SimpleNN()\n",
+    "criterion = nn.CrossEntropyLoss()\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Define a simple accuracy metric"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "b52198cc-39af-4a6f-b814-0f3d70eb6e7e",
+     "showTitle": true,
+     "title": "Define a simple accuracy metric"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Function to calculate accuracy\n",
+    "def calculate_accuracy(y_pred, y_true):\n",
+    "    _, predicted = torch.max(y_pred, 1)\n",
+    "    correct = (predicted == y_true).sum().item()\n",
+    "    accuracy = correct / y_true.size(0)\n",
+    "    return accuracy\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "4525800f-012c-4203-84d2-cb4017dc3d93",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "## Define the model signature\n",
+    "[Link to the mlflow.models.infer_signature docs](https://mlflow.org/docs/latest/python_api/mlflow.models.html#mlflow.models.infer_signature)\n",
+    "\n",
+    "The input should be one of the following:\n",
+    "- pandas.DataFrame\n",
+    "- pandas.Series\n",
+    "- dictionary of { name -> numpy.ndarray}\n",
+    "- numpy.ndarray\n",
+    "- pyspark.sql.DataFrame\n",
+    "- scipy.sparse.csr_matrix\n",
+    "- scipy.sparse.csc_matrix\n",
+    "\n",
+    "The output should be one of the following:\n",
+    "- pandas.DataFrame\n",
+    "- numpy.ndarray,\n",
+    "- Dict[str, numpy.ndarray]\n",
+    "- pyspark.sql.dataframe.DataFrame\n",
+    "\n",
+    "\n",
+    "\n",
+    "If you've already logged a model, you can add a signature to the logged model with [this API](https://www.mlflow.org/docs/2.8.0/models.html#set-signature-on-logged-model) as well."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "2b17c594-aa9a-4e0d-86fe-28d687dfa658",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "from mlflow.models.signature import infer_signature\n",
+    "\n",
+    "# Infer the signature of the model\n",
+    "sample_input = X_train_tensor[:1]\n",
+    "model.eval()\n",
+    "with torch.no_grad():\n",
+    "    sample_output = model(sample_input)\n",
+    "signature = infer_signature(sample_input.numpy(), sample_output.numpy())\n",
```

Comment:
We should also explain a bit about how `infer_signature` works and its limitation.